react/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { FC, useState, Fragment } from 'react'
+import { useQuery } from 'react-apollo'
+import { useIntl, defineMessages, FormattedMessage } from 'react-intl'
+import { Helmet, ExtensionPoint, useRuntime } from 'vtex.render-runtime'
+import { Button } from 'vtex.styleguide'
+import { usePWA } from 'vtex.store-resources/PWAContext'
+
+// import { Analytics } from './components/Analytics'
+import { OrderGroupContext } from './components/OrderGroupContext'
+import withoutSSR from './components/WithoutSSR'
+import ErrorMessage from './components/ErrorMessage'
+import GET_ORDER_GROUP from './graphql/getOrderGroup.graphql'
+import NotFound from './Icons/NotFound'
+import Forbidden from './Icons/Forbidden'
+import Skeleton from './Skeleton'
+import { CurrencyContext } from './components/CurrencyContext'
+import { Analytics } from './components/Analytics'
+
+interface OrderGroupData {
+  orderGroup: OrderGroup
+}
+
+const messages = defineMessages({
+  title: {
+    id: 'store/page.title',
+    defaultMessage: '',
+  },
+  invalidTitle: {
+    id: 'store/order.error.invalid.title',
+    defaultMessage: '',
+  },
+  invalidMessage: {
+    id: 'store/order.error.invalid.message',
+    defaultMessage: '',
+  },
+  notLoggedTitle: {
+    id: 'store/order.error.not-logged-in.title',
+    defaultMessage: '',
+  },
+  notLoggedMessage: {
+    id: 'store/order.error.not-logged-in.message',
+    defaultMessage: '',
+  },
+})
+
+const OrderPlaced: FC = () => {
+  const intl = useIntl()
+  const runtime = useRuntime()
+  const { settings = {} } = usePWA() || {}
+  const [installDismissed, setInstallDismissed] = useState(false)
+  const { data, loading, error } = useQuery<OrderGroupData>(GET_ORDER_GROUP, {
+    variables: {
+      orderGroup: runtime.query.og,
+    },
+  })
+
+  // render loading skeleton if query is still loading
+  if (loading) return <Skeleton />
+
+  // forbidden error
+  if (
+    error?.message.includes('403') ||
```

Comment:
Can you garantee that the only '403' in this error message is the status code?

Reply:
Yeah, did this a while ago. However, this can and should be improved in a near future.